src/NServiceBus.Core.Tests/Recoverability/SecondLevelRetries/SecondLevelRetriesTests.cs

```diff
@@ -190,7 +186,10 @@ IncomingMessage CreateMessage(string id = "id", string slrRetryHeader = null, by
         {
             var headers = string.IsNullOrEmpty(slrRetryHeader)
                 ? new Dictionary<string, string>()
-                : new Dictionary<string, string> { { Headers.Retries, slrRetryHeader }};
+                : new Dictionary<string, string>
```

Comment:
This is done by our template right? I usually prefer one liners for single entry dictionaries. But not a big deal